doc/user/content/sql/create-source/postgres.md

```diff
@@ -0,0 +1,139 @@
+**HOST** | Postgres host. See [Host options](#host-options) for details.
+**PUBLICATION** _publication_name_ | Postgres publication.
+
+### Host options
+
+Field | Type | Value
+------|------|------
+_host_  | Host for Postgres connection.
+_port_  | Port for Postgres connection.
+_user_  | User for Postgres connection.
+_password_  | Password for Postgres connection.
+_dbname_ | Postgres database name.
+_sslmode_ | Optionally specify TLS mode. Valid options are `disable`, `prefer`, `require`, `verify-ca`, and `verify-full`.
+_sslcert_ | SSL certificate.
+_sslkey_  |  SSL key.
+_sslrootcert_  | SSL root certificate.
+
+## Details
+
+### Materialized source details
+
+Materializing a source keeps data it receives in an in-memory
+[index](/overview/api-components/#indexes), the presence of which makes the
+source directly queryable. In contrast, non-materialized sources cannot process
+queries directly; to access the data the source receives, you need to create
+[materialized views](/sql/create-materialized-view) that `SELECT` from the
+source.
+
+For a mental model, materializing the source is approximately equivalent to
+creating a non-materialized source, and then creating a materialized view from
+all of the source's columns:
+
+```sql
+CREATE SOURCE src ...;
+CREATE MATERIALIZED VIEW src_view AS SELECT * FROM src;
+```
+
+The actual implementation of materialized sources differs, though, by letting
+you refer to the source's name directly in queries.
+
+For more details about the impact of materializing sources (and implicitly
+creating an index), see [`CREATE INDEX`: Details &mdash; Memory
+footprint](/sql/create-index/#memory-footprint).
+
+### PostgreSQL source details
+
+Materialize makes use of PostgreSQL's native replication capabilities to create a continuously updated replica of the desired Postgres tables.
+
+Before creating the source in Materialize, you must:
+
+- Set up your Postgres database to allow logical replication.
+- Ensure that the user for your Materialize connection has `REPLICATION` privileges.
+- Create a Postgres publication containing the data to be streamed to Materialize. Since Postgres sources are materialized (kept in memory) in their entirety, we strongly recommend that you limit publications only to the data you need to query.
+
+The materialized source created from the publication contains the raw data stream of replication updates, which you can then split into the familiar Postgres table view with [`CREATE VIEWS`](/sql/create-views/).
+
+Note that if you stop or delete Materialize without first dropping the Postgres source, the Postgres replication slot isn't deleted and will continue to accumulate data. In such cases, you may want to manually delete the Materialize replication slot in order to recover memory. Materialize replication slot names always begin with `materialize_` for easy identification.
+
+#### Restrictions on Postgres sources
```

Comment:
i think we still have the limitation that timestamptz columns cant be imported? https://github.com/MaterializeInc/materialize/issues/5971

Reply:
Makes sense -- as a start, we could just link to https://materialize.com/docs/sql/types/ ?

I specifically called out timestamptz because it's a popular column type that Materialize _does_ support but Postgres sources don't support

Reply:
Then we do support it :) I think you must be remembering some very early iteration of pg sources that indeed had this limitation. The released version supports all the datatypes that we support